src/components/form-elements/checkbox-radio.html

```diff
@@ -25,9 +25,9 @@ <h3 class="docs-block__header">Checkboxes</h3>
                 <div class="sg-checkbox">
                     <input class="sg-checkbox__element" type="checkbox" id="checkbox-3">
                     <label class="sg-checkbox__ghost" for="checkbox-3">
-                        <svg class="sg-icon sg-icon--adaptive sg-icon--x8">
-                            <use xlink:href="#icon-check"></use>
-                        </svg>
+                      <svg class="sg-icon sg-icon--adaptive sg-icon--x10">
```

Comment:
So when migrating we will have to change it everywhere :O 
Please make it as major update